addon/templates/components/freestyle/bootstrap-pagination-nav.hbs

```diff
@@ -1,9 +1,19 @@
 {{#section.subsection name="Pagination"}}
+  {{#freestyle-annotation}}
+    {{bootstrap-input
+      label='Page Size Option'
+      value=pageSize}}
+    {{bootstrap-checkbox
+      isChecked=showSinglePage
+      label='Show Single Page Option'
+      onclick=(action (mut showSinglePage))}}
+  {{/freestyle-annotation}}
   {{#freestyle-usage "pagination" title="Pagination"}}
     {{bootstrap-pagination-nav
+      showSinglePage=showSinglePage
```

Comment:
I feel like this whole thing could have been solved with:
```
{{#if (gt totalPages 1)}}
  {{bootstrap-pagination-nav
    pageNumber=pageNumber
    pageSize=pagesize
    totalPages=totalPages
    moveToPage=(action (mut pageNumber))
  }}
{{/if}}
```

And that this is just adding code to support something that can already be done. Thoughts?

Reply:
Updating it to remove option. I am adding this snippet for this in the readme.